lib/telegram/bot/api/endpoints.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+module Telegram
+  module Bot
+    class Api
+      ENDPOINTS = {
+        ## `nil` for raw `result`
+        'getUpdates' => Types::Array.of(Types::Update),
+        'setWebhook' => Types::Bool,
```

Comment:
API doc says "True for success", I don't think we can receive `result: false`, `ok: false` I think will be without `result` at all, but I've decided to not force `Types::Strict::True` here.